project/plugins.sbt

```diff
@@ -3,14 +3,21 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
   Resolver.ivyStylePatterns
```

Comment:
Please update the name on line 2 to `HMRC-open-artefacts-ivy2` as per [here](https://github.com/hmrc/sbt-auto-build#usage). Thanks